docs/UserGuide.md

```diff
@@ -223,19 +263,36 @@ Examples:
 *  `list` followed by `edit 1 p/12345678 animal/Bob` edits the phone number and the pet name of the 1st fosterer in the address book to 12345678 and Bob respectively.
 *  `list` followed by `edit 2` changes the view to the profile page of the 2nd fosterer in the address book since parameters are not provided.
 
-Expected Output (success):
+Expected output (if successful):
+Format 1: 
+```agsl
+Edited Fosterer: [NAME]; Phone: [NUMBER]; Email: [EMAIL]; Address: [ADDRESS]; Housing: [HOUSING]; Availability: [AVAILABILITY]; Animal name: [ANIMAL NAME]; Animal type: [ANIMAL TYPE]; Tags: [TAG]
 ```
-Foster family details successfully edited!
+*  If a field is not edited, the output shows the fosterer’s original data of the field.
+
+Format 2: 
+```agsl
+Viewing Fosterer: [NAME]; Phone: [NUMBER]; Email: [EMAIL]; Address: [ADDRESS]; Housing: [HOUSING]; Availability: [AVAILABILITY]; Animal name: [ANIMAL NAME]; Animal type: [ANIMAL TYPE]; Tags: [TAG]
 ```
+*  The UI generates a pop-up window where details of the fosterer are shown.
```

Comment:
wait i thought a popup window is like a small window (like help) or smth HAHAHA but i might be wrong